src/view.tree.js

```diff
@@ -93,9 +93,25 @@ class TreeView {
     return str && str.replace(/[<>'"&]/g, '-');
   }
 
+  cmpVersions(a, b) {
+    let i, diff;
```

Comment:
Don't declare loop variable outside. Please read existing code so that you follow the same style instead trying a new style.